swift/apple/FirezoneKit/Sources/FirezoneKit/Views/MenuBar.swift

```diff
@@ -42,28 +42,12 @@ public final class MenuBar: NSObject {
 
     if let button = statusItem.button {
       button.image = signedOutIcon
-      button.target = self
-      button.action = #selector(statusItemClicked)
     }
 
     createMenu()
     setupObservers()
   }
 
-  @objc private func statusItemClicked() {
```

Comment:
So it turns out we can remove this since:

- we aren't customizing the menu anymore (the highlight bug doesn't happen)
- I added `NSApp.activate()` to the `sign_in` action to keep the bug fixed